src/main/java/collections/myHashMap/MyHashMap.java

```diff
@@ -0,0 +1,117 @@
+package collections.myHashMap;
+import collections.MyLinkedList;
+
+import java.util.LinkedList;
+import java.util.Objects;
+
+public class MyHashMap<K, V> {
```

Comment:
The generic type parameters here are redundant. In fact, this is not a Map of two parameters, but the `collections.myHashMap.Key` and `collections.myHashMap.Value`